app/controllers/posts_controller.rb

```diff
@@ -1,6 +1,6 @@
 class PostsController < ApplicationController
-  before_action :find_user, only: %i[index show like unlike]
-  before_action :find_post, only: %i[show like unlike]
+  before_action :find_user, only: %i[index show like unlike destroy]
+  before_action :find_post, only: %i[show like unlike destroy]
 
   def index
     page = params[:page] || 1
```

Comment:
- [ ] It is a requirement that you use CanCanCan for post authorization in the Post controller.  Currently you are relying on the current_user method to identify the currently authenticated user, and then it checks whether the current user is the owner of the post before performing editing or deleting the post. Kindly fix this.

e.g
```
  class PostsController < ApplicationController
    load_and_authorize_resource


  end
```